plugins/source/aws/resources/services/cloudwatchlogs/metric_filters_fetch.go

```diff
@@ -28,15 +26,3 @@ func fetchCloudwatchlogsMetricFilters(ctx context.Context, meta schema.ClientMet
 	}
 	return nil
 }
```

Comment:
nit: I think we are trying to not have a separate file for `fetch` functions and just put it all in the same file where the table is defined